src/sentry/issues/mark_reviewed.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from typing import Any, Dict, List
+
+from sentry.models import Group, Project, User
+from sentry.models.groupinbox import (
+    GroupInboxReason,
+    GroupInboxRemoveAction,
+    add_group_to_inbox,
+    remove_group_from_inbox,
+)
+from sentry.signals import issue_mark_reviewed
+
+
+def mark_reviewed(
+    in_inbox: bool,
+    group_list: List[Group],
+    project_lookup: Dict[int, Project],
+    acting_user: User | None,
+    http_referrer: str,
+    sender: Any,
+) -> bool:
+    """
+    Support moving groups in or out of the inbox via the Mark Reviewed button.
+
+    Returns the new inbox state.
```

Comment:
```suggestion
    Returns a boolean indicating whether or not the groups are now in the inbox.
```

Question, though: why are we returning it if we're also passing it in, and not changing it in the meantime? Just to kill two birds with one stone where we're calling it?